content/source/docs/extend/sdkv2-intro.html.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: "extend"
+page_title: "Home - Plugin Development: SDKv2"
+description: |-
+  Learn about a framework you can use to develop Terraform plugins.
+---
+
+# Terraform Plugin SDKv2
+
+Terraform Plugin SDKv2 is a precursor to the [Terraform Plugin Framework](/docs/plugin/framework/index.html) for developing Terraform plugins.
+
+~> **Important**: If you maintain a large existing provider, we recommend that you continue using SDKv2 to develop new resources and data sources. [Which SDK Should I Use?](/docs/plugin/which-sdk.html) explains the differences between SDKv2 and the framework.
```

Comment:
I think this is more aggressive than we'd like to be at this time. @aareet and @redeux can speak with more authority on this, but I think we're trying to position SDKv2 as the main SDK at the moment, with the framework being the early-adopter next version until the framework is more capable.

Reply:
Noted! Check the updated file and see if you think that's better :)